circle.yml

```diff
@@ -5,8 +5,27 @@ test:
   pre:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
-    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make build-server ;; esac:
-        parallel: true
+    - ? |
+          case $CIRCLE_NODE_INDEX in
+            0) NODE_ENV=test make build-server;
+               ;;
+
+            1) I18N_DIR=$CIRCLE_ARTIFACTS/translate;
+               mkdir -p $I18N_DIR;
+               make translate;
+               mv calypso-strings.pot $I18N_DIR/calypso-strings.pot;
+               if [[ $CIRCLE_BRANCH != "master" ]]; then
+                 git merge-base --fork-point master | xargs git checkout;
```

Comment:
If we were worried about the extra work, we could cache the results along with the base SHA1 so we can regenerate on rebase. Seems like more complexity than it's worth, though, imho.

We could also curl the artifact if we knew the build number, but I can't see any way to connect sha1 to build number.